.github/workflows/auto-update-pr.yaml

```
@@ -0,0 +1,14 @@
+name: auto-update-pr
+on:
+  push: {}
+jobs:
+  autoupdate:
+    name: Auto update PR
+    runs-on: ubuntu-22.04
```

Is there a reason to tie this workflow to that Ubuntu version?
```suggestion
    runs-on: ubuntu-latest
```